3g-watchdog

```diff
@@ -30,18 +31,21 @@ TEST_HOSTS = ["8.8.8.8", "8.8.4.4"]
 TEST_INTERVAL_SECS = 5 * 60
 
 ATTINY_ADDRESS = 0x04
-MODEM_PIN = 18
 
+def parse_config():
+    with open("/etc/cacophony/3g-watchdog.yaml", 'r') as stream:
+        config = yaml.load(stream)
```

Comment:
It's best to use `yaml.safe_load` to avoid the possibility of arbitrary code execution